lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/array_item_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if array_items["type"] == "string" %>
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: field_name.humanize,
+    },
+    name:,
+    id:,
+    value: field_value,
+  } %>
+<% elsif array_items["type"] == "object" %>
```

Comment:
I'm not sure I quite understand how validation is working, but I noticed if I put an incorrect value into the telephone `type` input I don't get a form validation error at the point of save and continue - I would have thought it should validate it against the schema then and there? Could be something we pick up in another PR but just flagging

Reply:
Yep, good idea. I'll add that as a separate task